english/the-slayer/marshmallow-slayer.md

```diff
@@ -0,0 +1,5 @@
+Once upon a time it was the most terrible marshmallow, he slain too many of poor little marshmallows.
+But for fortune it was a heroic marshmallow called Freddie. Lets look at how he saved the little
+marshmallows:
+
+[Freddie](the-slayer/freddie/freddie.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.